doc/man1/openssl-dgst.pod.in

```diff
@@ -30,7 +30,9 @@ B<openssl> B<dgst>|I<digest>
 [B<-macopt> I<nm>:I<v>]
 [B<-fips-fingerprint>]
 {- $OpenSSL::safe::opt_engine_synopsis -}
+{- output_off() if $disabled{"deprecated-3.0"}; "" -}
 [B<-engine_impl> I<id>]
+{- output_on() if $disabled{"deprecated-3.0"}; "" -}
```

Comment:
Unfortunately, the perl bits do not eat up the line ending that follows it, so you end up getting the `-engine_impl` in a paragraph of its own (empty line before and after).  The easiest trick to resolve that is this:

```
{- output_off() if $disabled{"deprecated-3.0"}; ""
-}[B<-engine_impl> I<id>]{-
   output_on() if $disabled{"deprecated-3.0"}; "" -}
```

It's not the best-looking thing in the world, but it does the job.

For larger chunks of text that are separate paragraphs anyway, this is less of a worry.

(this is a fallacy with Text::Template, but it's otherwise simple and small and suitable for our purposes...  there are other template toolkits that do this kind of stuff better, but they are also quite large)

Reply:
Done here and in openssl-list.pod.in.